### PR TITLE
Docker b**ches 🐳

### PR DESCRIPTION
**Pull request by ogizanagi**
🐳

### CODE REVIEW

**Comment by nervo** [comment]
'just fixed some behaviours :)

**Review by nervo — APPROVED** [review]
Oh, that's the way, uh-huh uh-huh
I like it, uh-huh, uh-huh

**Comment by ogizanagi** [comment]
Thanks for the fixes :)

File: vars/main.yml [hunk 1]
```diff
@@ -9,6 +9,17 @@ manala_skeleton_patterns:
...
 
   manala: &pattern_manala
+    # Accounts
+    accounts_groups: "{{
+      (manala_skeleton_options.docker)|ternary([{'group': 'docker'}],[])
+    }}"
+    accounts_users_groups_exclusive: false
+    accounts_users:
+      - user:   "{{ manala_skeleton_options.app.user }}"
+        groups: ['docker']
```
Comment (ogizanagi) [inline vars/main.yml:19]: Oops. This should have been removed, isn't?

**Comment by nervo** [comment]
Oooh, @chalasr is alive !
:car: :car: :car: :boom: :car: :car:

File: vars/main.yml (hunk 1 above)
Comment (nervo) [reply vars/main.yml:19]: Ach', shit !

Comment (nervo) [reply vars/main.yml:19]: Let's ninja commit on master !

Comment (ogizanagi) [reply vars/main.yml:19]: No one will see. 🐱‍👤  

:shipit:

Comment (nervo) [reply vars/main.yml:19]: I was pretty sure there was a :ninja: emoji... Anyway: https://github.com/manala/ansible-role-skeleton/commit/1a9829540283993cd497d028384671d74fe60e90

Comment (ogizanagi) [reply vars/main.yml:19]: http://emojipedia.org/ninja-cat/ as a workaround 😅

Comment (nervo) [reply vars/main.yml:19]: Neeeeeeeeed !

**Pull request merged**